js/search.js

```diff
@@ -102,7 +115,10 @@
 			e.preventDefault();
 			if ($(e.currentTarget).is('.disabled, :disabled')) return;
 
-			if(this.$element.hasClass('searched')) {
+			if(this.$element.hasClass('pending')) {
```

Comment:
Instead of an if/else here, you can just combine them into a single `||` statement in the if expression.